elyra/metadata/schemas/kfp.json

```diff
@@ -47,7 +47,7 @@
           "title": "Kubeflow Pipelines API Endpoint",
           "description": "The Kubeflow Pipelines API endpoint",
           "type": "string",
-          "pattern": "(?:(?:http|https)://)?(www\\.)?[a-zA-Z0-9@:%._\\+~#?&//=]{2,256}\\.[a-z]{2,6}\\b([-a-zA-Z0-9@:%._\\+~#?&//=]*)",
+          "pattern": "^((?:https?:\\/\\/)?[^./]+(?:\\.[^./]+)+(?:\\/.*)?)$",
```

Comment:
yes, this looks much better than the previous regex!

IIUC, looking at the structure of error message in https://issues.redhat.com/browse/RHOAIENG-5071, the validation is performed at

https://github.com/opendatahub-io/elyra/blob/3f11b019939effbac016862f8caffda8f560e724/elyra/metadata/manager.py#L159

Reply:
This regexp is great, though we dont need it for the kubeflow pipeline endpoint only for the cos_endpoint that is below, can you revert this line back please.